### DISCUSSION TITLE
Generate Change Log

### DISCUSSION THREAD

**Discussion by blink1073** [community/community#22861] [New to GitHub]
<p>Idea: A way to generate a change log for a Milestone.  It would use a template file included in the repo itself, and allow grouping by labels.  This would be a huge boon for OSS maintainers.  If it were shown in a text buffer, it could be pasted into a GH release or in a local change log addendum.</p>

**Comment by afshin** [comment]
<p>We feel this pain with each release of <a href="https://github.com/jupyterlab/jupyterlab" rel="nofollow noopener">JupyterLab</a> because it is a large project with lots of contributors. A head start on the change log would be very helpful.</p>

**Comment by angela-crist** [comment, marked as answer]
<p>Hi <a class="mention" href="/blink1073">@blink1073</a>,</p>
<p>Thanks for this feedback! We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Please let me know if you have any other questions.</p>
<p>Also, this post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our community organized so users can more easily find information.</p>
<p>As you’ll notice, your topic is now here in the How to use Git and GitHub board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Cheers!</p>

**Comment by blink1073** [comment]
<p>Hi Angela, thank you for your consideration!</p>
